string[] classes_to_find
---
Header header
geometry_msgs/Point[] object_coordinates
